static/src/javascripts/projects/commercial/modules/prebid/slot-config.js

```diff
@@ -59,6 +64,22 @@ const getSlots = (contentType: string): Array<PrebidSlot> => {
             key: 'comments',
             sizes: [[160, 600], [300, 250], [300, 600]],
         },
+        // Interactive banner to work with
+        // the incorrect 'dfp-ad--N' adslot naming scheme.
+        // Example on:
+        // https://www.theguardian.com/us-news/ng-interactive/2018/nov/06/midterm-elections-2018-live-results-latest-winners-and-seats
+        {
+            key: 'banner',
+            matchesDfpSlot: (dfpa: Advert) =>
```

Comment:
This seems to be mixing up definitions with logic. The fact that we have hard-coded class names here strikes me as something that may cause us problems down the line.

Reply:
the 'key' is also hardcoded, as the matching relies on the assumption the GTP adslots would follow the right naming scheme, with a part of the GTP slot ID matching with the 'key' property of these objects.

 Hopefully we can get rid of 'matchesDfpSlot' in the near future (will file a card to address that) when the interactive GTP slots follow this convention. Until this happens we need to guarantee that we can associate Prebid slots to GTP Adslots in a flexible way to avoid missing a revenue opportunity.